mmv1/products/netapp/activeDirectory.yaml

```diff
@@ -0,0 +1,187 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'activeDirectory'
+description: |
+  ActiveDirectory is the public representation of the active directory config.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/netapp/volumes/docs/configure-and-use/active-directory/about-ad'
+  api: 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.activeDirectories'
+base_url: projects/{{project}}/locations/{{location}}/activeDirectories
+self_link: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+create_url: projects/{{project}}/locations/{{location}}/activeDirectories?activeDirectoryId={{active_directory_id}}
+create_verb: :POST
+update_url: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+update_verb: :PATCH
+update_mask: true
+delete_url: projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}
+delete_verb: :DELETE
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+id_format: 'projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}'
+import_format: ['projects/{{project}}/locations/{{location}}/activeDirectories/{{active_directory_id}}']
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Name of the region for the policy to apply to.
+  - !ruby/object:Api::Type::String
+    name: 'active_directory_id'
+    description: |
+      The resource name of the Active Directory pool. Needs to be unique per location.
+    required: true
+    immutable: true
+    url_param_only: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'netapp_active_directory_basic'
+    primary_resource_id: 'test_active_directory_basic'
+    skip_test: true
+    vars:
+      active_directory_basic_name: 'test-active-directory-basic'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'netapp_active_directory_full'
+    primary_resource_id: 'test_active_directory_full'
+    skip_test: true
+    vars:
+      active_directory_full_name: 'test-active-directory-full'
+properties:
+  # Fields go here
+  - !ruby/object:Api::Type::String
+    name: name
```

Comment:
Not sure what this is good for. "name" is the resourceID of this resource, as suggested above. IMHO this parameter here needs to get deleted.